.changeset/tough-ears-suffer.md

```diff
@@ -0,0 +1,7 @@
+---
+'@envelop/types': minor
+'@envelop/core': minor
+'@envelop/instruments': major
+---
+
+Add new Tracer API
```

Comment:
It's called "instruments" now, right?

Also, please add some more information and an example to the changelog - lets increase awareness.